src/chapters/vstools/studio.rst

```diff
@@ -0,0 +1,22 @@
+If It Ain't Broke, Add a Breakpoint!
+====================================
+
+On your machine, open up your fork of the ``HelloASPDotNETDemo`` `repository <https://github.com/LaunchCodeEducation/HelloASPDotNETDemo>`_ in Visual Studio.
+The purpose of this studio is to talk about your current debugging strategies and how to make the most of the debugger tools discussed in this chapter.
+
+Before we start practicing with debugging tools, go over with the group one error you encountered when working on your own version of ``HelloASPDotNET``.
+
+#. What was the error?
+#. How did you solve this error? What have been the strategies and tools you have been using so far to debug your code?
+#. Could one of the debugging tools helped you when addressing this error?
+
+Review the code from ``HelloASPDotNETDemo`` and use the debugging tools in Visual Studio to address potentional problems.
```

Comment:
For this section, im having trouble envisioning many students actually doing this if we don't provide specific items to look for. I think we talked about like suggesting lines to add breakpoints and looking for specific variables? adding a certain watch someplace maybe?

Reply:
That escaped my mind! I went back and added a direction to get them started and an example to one of the questions so that discussion can flow more easily! Thank you!